packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -243,6 +247,47 @@ class RenderListWheelViewport
     _hasScrolled();
   }
 
+  /// Visual effect based on how we see the wheel. Imagine we are at some
```

Comment:
Reading this it's easy to understand the effect from a relative sense but it's hard to predict the effect from an absolute sense. Can we make it easier to quantify this effect (either by making equivalents in the physical world etc)? In other word, how can I mentally visualize what would happen if I put in `3.0`?.